docs/data/joy/getting-started/templates/TemplateCollection.js

```diff
@@ -51,6 +51,10 @@ const AUTHORS = {
     name: 'MUI',
     link: 'https://twitter.com/MUI_hq',
   },
+  'rental-dashboard': {
+    name: 'Steve Ernstberger',
+    link: 'https://twitter.com/MUI_hq',
+  },
```

Comment:
Heya @sernstberger, this is amazing ⎯ thanks for putting in the effort! 🙌 
Would you mind pasting the reference Untitled UI on the `DESIGNS` const below?

Reply:
Just added. Thanks.